src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -2712,7 +2712,7 @@ private static bool ContainsModifier(SyntaxTokenList modifiers, SyntaxKind modif
             }
             else
             {
-                membersByName = membersAndInitializers.NonTypeMembers.ToDictionary(s => s.Name, StringOrdinalComparer.Instance);
+                membersByName = membersAndInitializers.NonTypeMembers.ToDictionary(static s => s.Name);
```

Comment:
were using a comparer.  but not with the dictionary above.  seems like that couldn't possibly ever have been intentional.  so removed.